client/js/lounge.js

```diff
@@ -79,18 +79,45 @@ $(function() {
 		if (target.hasClass("user")) {
 			output = templates.contextmenu_item({
 				class: "user",
+				action: "whois",
 				text: target.text(),
 				data: target.data("name"),
 			});
+			output += templates.contextmenu_divider();
+			output += templates.contextmenu_item({
+				class: "action-whois",
```

Comment:
Why are we duplicating this action? If this is for clarity sake (which would be a valid argument I think), then really we should make the first entry non-clickable. Visually different line for the name, à la Slack, minus the avatar picture:

<img width="333" alt="screen shot 2017-12-10 at 23 43 08" src="https://user-images.githubusercontent.com/113730/33815999-eed21720-de03-11e7-808c-a89f1b2ae37d.png">

Reply:
Left the duplicate activity because long time lounge users might expect that action. Also it doesn't hurt.

Reply:
Well, it's going literally 5mm lower, so it's not too confusing either :)
I think we should explore better UI ideas for this, but it doesn't have to be done in your PR.